src/algebra/ordered_pi.lean

```diff
@@ -19,6 +19,33 @@ variables (x y : Π i, f i) (i : I)
 
 namespace pi
 
+/-- The product of a family of ordered commutative monoids is an ordered commutative monoid. -/
+@[to_additive "The product of a family of ordered additive commutative monoids is
+  an ordered additive commutative monoid."]
+instance ordered_comm_monoid {ι : Type*} {Z : ι → Type*} [∀ i, ordered_comm_monoid (Z i)] :
+  ordered_comm_monoid (Π i, Z i) :=
+{ mul_le_mul_left := λ f g w h i, mul_le_mul_left' (w i) _,
```

Comment:
I think there are some instances below that reprove this field; can you use `..pi.ordered_comm_monoid` in those places instead?

Reply:
Done.